src/components/DetailSnippet/TitleHashtag/TitleHashtag.js

```diff
@@ -0,0 +1,36 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const TitleWrapper = styled.div`
+  height: 55px;
+  padding: 5px 0px 0px 10px;
+`;
+
+const Hashtag = styled.a`
+  margin-right: 15px;
+  color: #6A74CD;
+  font-size: 45px;
+  font-weight: bold;
+  text-decoration: none;
+`;
+
+export default function TitleHashtag({ hashtags }) {
+  return (
+    <TitleWrapper>
+      {hashtags.map((hashtag) => {
+        return (
+          <Hashtag
+            key={hashtag}
+            href={`/?search=${hashtag}`}
+          >
+            {hashtag}
+          </Hashtag>
+        );
+      })}
```

Comment:
스타일 통일을 위해 수정 부탁드립니다!

```suggestion
      {hashtags.map((hashtag) => (
        <Hashtag
          key={hashtag}
          href={`/?search=${hashtag}`}
        >
          {hashtag}
        </Hashtag>
      ))}
```